sips/slip-template.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: slip
+disqus: true
+title: SLIP-NN - SLIP Title
```

Comment:
This is super valuable. I think its existence would lead to better proposals/slips just by virtue of outlining a set of good sections.